tests/integration/hub_usage/dummyhub_abs/__init__.py

```diff
@@ -1,4 +1,4 @@
-from helper import foo
+from .helper import foo
```

Comment:
I think the point of this file is to have this and fail for that reason

Reply:
okay I'll revert this change